### ISSUE TITLE
Reconcile not signed up user's screenshots on signup

### ISSUE THREAD

**Issue by JonathanMH** [HackYourFuture-CPH/FP-annotatetheweb-class10#85]
We need to make sure that the screenshots created by a user before their signup, remains theirs to edit and delete when they create a user account / register. 

cc: @bytewiz Do you think it's possible to get an anonymous token through firebase auth that we later can assign an email address or a facebook user to?

This issue depends on #83

**Comment by bytewiz** [comment]
@JonathanMH Yes, that is possible 👍

**Comment by zkwsk** [comment]
Note that this one is low priority. We need to get the core functionality running first.
